File: PI3A16/src/EventBundle/Entity/Event.php
<?php

namespace EventBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Event
 *
 * @ORM\Table(name="event")
 * @ORM\Entity(repositoryClass="EventBundle\Repository\EventRepository")
 */
class Event
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;
    /**
     *@ORM\ManyToOne(targetEntity="UserBundle\Entity\User")
     * @ORM\JoinColumn(name="idorganisateur", referencedColumnName="id")
     */

    private $idorganisateur;

    /**
     *@ORM\ManyToOne(targetEntity="EventBundle\Entity\eventtype")
     * @ORM\JoinColumn(name="eventtype", referencedColumnName="id")
     */

    private $eventtype;

    /**
     * @var string
     *
     * @ORM\Column(name="name", type="string", length=255)
     */
    private $name;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=255)
     */
    private $description;

    /**
     * @var int
     *
     * @ORM\Column(name="capacity", type="integer")
     */
    private $capacity;

    /**
     * @var int
     *
     * @ORM\Column(name="reserved", type="integer")
     */
    private $reserved;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="datetime")
     */
    private $date;

    /**
     * @var int
     *
     * @ORM\Column(name="numberofdays", type="integer")
     */
    private $numberofdays;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set name
     *
     * @param string $name
     *
     * @return Event
     */
    public function setName($name)
    {
        $this->name = $name;

        return $this;
    }

    /**
     * Get name
     *
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Event
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set capacity
     *
     * @param integer $capacity
     *
     * @return Event
     */
    public function setCapacity($capacity)
    {
        $this->capacity = $capacity;

        return $this;
    }

    /**
     * Get capacity
     *
     * @return int
     */
    public function getCapacity()
    {
        return $this->capacity;
    }

    /**
     * Set reserved
     *
     * @param integer $reserved
     *
     * @return Event
     */
    public function setReserved($reserved)
    {
        $this->reserved = $reserved;

        return $this;
    }

    /**
     * Get reserved
     *
     * @return int
     */
    public function getReserved()
    {
        return $this->reserved;
    }

    /**
     * Set date
     *
     * @param \DateTime $date
     *
     * @return Event
     */
    public function setDate($date)
    {
        $this->date = $date;

        return $this;
    }

    /**
     * Get date
     *
     * @return \DateTime
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * Set numberofdays
     *
     * @param integer $numberofdays
     *
     * @return Event
     */
    public function setNumberofdays($numberofdays)
    {
        $this->numberofdays = $numberofdays;

        return $this;
    }

    /**
     * Get numberofdays
     *
     * @return int
     */
    public function getNumberofdays()
    {
        return $this->numberofdays;
    }

    /**
     * Set idorganisateur
     *
     * @param \UserBundle\Entity\User $idorganisateur
     *
     * @return Event
     */
    public function setIdorganisateur(\UserBundle\Entity\User $idorganisateur = null)
    {
        $this->idorganisateur = $idorganisateur;

        return $this;
    }

    /**
     * Get idorganisateur
     *
     * @return \UserBundle\Entity\User
     */
    public function getIdorganisateur()
    {
        return $this->idorganisateur;
    }

    /**
     * Set eventtype
     *
     * @param \EventBundle\Entity\eventtype $eventtype
     *
     * @return Event
     */
    public function setEventtype(\EventBundle\Entity\eventtype $eventtype = null)
    {
        $this->eventtype = $eventtype;

        return $this;
    }

    /**
     * Get eventtype
     *
     * @return \EventBundle\Entity\eventtype
     */
    public function getEventtype()
    {
        return $this->eventtype;
    }
}

File: PIDEV/src/entities/event.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.sql.Date;
import java.sql.Time;


/**
 *
 * @author zaxs
 */
public class event {
    private int id;
    private String idorganisateur;
    private String eventtype;
    private String name;
    private String description;
    private int capacity;
    private int reserved;
    private Date date;
    private int numberOfDays;
    private Time time;
    private String adresse;
    private String etat;
    private String ville;
    private int codePostal;
    private boolean free;
    private Float prix;

    public event(int id, String idorganisateur, String eventtype, String name, String description, int capacity, int reserved, Date date, int numberOfDays, Time time, String adresse, String etat, String ville, int codePostal, boolean free, Float prix) {
        this.id = id;
        this.idorganisateur = idorganisateur;
        this.eventtype = eventtype;
        this.name = name;
        this.description = description;
        this.capacity = capacity;
        this.reserved = reserved;
        this.date = date;
        this.numberOfDays = numberOfDays;
        this.time = time;
        this.adresse = adresse;
        this.etat = etat;
        this.ville = ville;
        this.codePostal = codePostal;
        this.free = free;
        this.prix = prix;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getIdorganisateur() {
        return idorganisateur;
    }

    public void setIdorganisateur(String idorganisateur) {
        this.idorganisateur = idorganisateur;
    }

    public String getEventtype() {
        return eventtype;
    }

    public void setEventtype(String eventtype) {
        this.eventtype = eventtype;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public int getReserved() {
        return reserved;
    }

    public void setReserved(int reserved) {
        this.reserved = reserved;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public int getNumberOfDays() {
        return numberOfDays;
    }

    public void setNumberOfDays(int numberOfDays) {
        this.numberOfDays = numberOfDays;
    }

    public Time getTime() {
        return time;
    }

    public void setTime(Time time) {
        this.time = time;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public String getEtat() {
        return etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }

    public String getVille() {
        return ville;
    }

    public void setVille(String ville) {
        this.ville = ville;
    }

    public int getCodePostal() {
        return codePostal;
    }

    public void setCodePostal(int codePostal) {
        this.codePostal = codePostal;
    }

    public boolean isFree() {
        return free;
    }

    public void setFree(boolean free) {
        this.free = free;
    }

    public Float getPrix() {
        return prix;
    }

    public void setPrix(Float prix) {
        this.prix = prix;
    }

    
    
}
